dev/RadioButtons/RadioButtons.idl

```diff
@@ -14,8 +14,8 @@ unsealed runtimeclass RadioButtons : Windows.UI.Xaml.Controls.Control
     Windows.Foundation.Collections.IVector<Object> Items{ get; };
     Windows.UI.Xaml.DataTemplate ItemTemplate;
```

Comment:
Would it make sense to change ItemTemplate to object so you can support DataTemplateSelectors as well. In WinUI3, this probably should be IElementFactory

Reply:
Hmm right now this is actually unused.  I think I just never actually noticed it. I wonder if better would be to delete it.

---
In reply to: [347397126](https://github.com/microsoft/microsoft-ui-xaml/pull/1623#discussion_r347397126) [](ancestors = 347397126)